src/drivers/mct/cime_config/buildlib_cmake

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
```

Comment:
I don't think that we need to have all of these buildlib_cmake files - they can be combined with the existing buildlib files.

Reply:
Good point.

Reply:
This can be a refactor for a future time.